addons/apecloud-mysql/0.6.3.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: extensions.kubeblocks.io/v1alpha1
+kind: Addon
+metadata:
+  name: apecloud-mysql
+  annotations:
+    addons.extensions.kubeblocks.io/default-is-empty: "true"
+    meta.helm.sh/release-name: kubeblocks
+    meta.helm.sh/release-namespace: kb-system
+    xx.kubeblocks.io/kubeblocks-version: ">= 0.6.3 && < 0.7.0-beta.0 "
+    xx.kubeblocks.io/model: RDBMS
+  labels:
+    app.kubernetes.io/instance: kubeblocks
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: kubeblocks
+    app.kubernetes.io/version: 0.6.3
+    helm.sh/chart: kubeblocks-0.6.3
+    kubeblocks.io/provider: community
```

Comment:
```
  name: apecloud-mysql
  annotations:
    addons.extensions.kubeblocks.io/default-is-empty: "true"
    addons.extensions.kubeblocks.io/kubeblocks-version: ">= 0.6.3 && < 0.7.0-beta.0 "
    addons.extensions.kubeblocks.io/model: RDBMS
  labels:
    app.kubernetes.io/managed-by: kubeblocks
    app.kubernetes.io/name: apecloud-mysql
    app.kubernetes.io/version: 0.6.3
    kubeblocks.io/provider: apecloud
```

The helm annotations and labels are not needed, that are added by helm. And `app.kubernetes.io/managed-by: kubeblocks` and `app.kubernetes.io/name: apecloud-mysql` should be added by addon controller, do not necessary in addon labels.